### DISCUSSION TITLE
Welcome to Flask-Discord Discussions!

### DISCUSSION THREAD

**Discussion by weibeu** [weibeu/Flask-Discord#35] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ZoutigeWolf** [comment]
How can i change the scopes on `discord.create_session()`

**Reply by ZoutigeWolf** [reply]
nevermind im blind lol

**Comment by pr0m** [comment]
How to check role of a user in a specific guild ? Should I use a bot ? (Required to access BOT resources)

**Reply by weibeu** [reply]
Yeah I think you will have to use [GET /guilds/{guild.id}/members/{user.id}](https://discord.com/developers/docs/resources/guild#get-guild-member) endpoint as I don't think this is provided as OAuth2 resource.
There is this util you can use: [`DiscordOAuth2Session.bot_request`](https://flask-discord.readthedocs.io/en/latest/api.html#flask_discord.DiscordOAuth2Session.bot_request)

**Reply by pr0m** [reply]
Perfect it's works thank you

**Comment by Killezz** [comment]
Hi. How to make user to be logged in also after closing and opening browser again? Right now I have to login again everytime. Thanks in advance.

**Reply by weibeu** [reply]
You can set `flask.session.permanent = True`. For more reference, check [`flask.session.permanent`](https://flask.palletsprojects.com/en/2.0.x/api/#flask.session.permanent).
